### PR TITLE
 [ENG-511]  Alert event extends transaction

### PR DESCRIPTION
**Pull request by delta003**

### CODE REVIEW

File: packages/actions-test/src/event.ts [hunk 1]
```diff
@@ -53,6 +55,8 @@ export class TestTransactionEvent implements TransactionEvent {
         this.input = "0x"
         this.value = "0x"
         this.nonce = "0x"
+        this.transactionHash = "0x"
+        this.alertId = "-"
```
Comment (BogdanHabic) [inline packages/actions-test/src/event.ts:59]: Maybe generate a fake uuid? I'm not sure what the zero value would be here.

**Review by BogdanHabic — DISMISSED** [review]

**Comment by delta003** [comment]
LGTM

File: packages/actions-test/src/event.ts [hunk 2]
```diff
@@ -69,13 +73,31 @@ export class TestLog implements Log {
 }
 
 export class TestAlertEvent implements AlertEvent {
```
Comment (delta003) [inline packages/actions-test/src/event.ts:75]: I think this was added very recently and not used, so we can clear up TestAlertEvent. TestTransactionEvent should be used, if necessary.

File: packages/actions-test/src/event.ts (hunk 1 above)
Comment (delta003) [reply packages/actions-test/src/event.ts:59]: Honestly, I don't think anything depends on alertId here except our FE attempting to parse it and link so we can remove this from API, and just keep it as parameter in payload (keep it in Conjure TransactionPayload, but not here in TransactionEvent)

**Review by delta003 — COMMENTED** [review]
LGTM

**Pull request merged**